specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2024-09-01-preview/appconfiguration.json

```diff
@@ -0,0 +1,3168 @@
+          "readOnly": true
+        },
+        "readOnly": {
+          "description": "Whether this key can only be used for read operations.",
+          "type": "boolean",
+          "readOnly": true
+        }
+      }
+    },
+    "RegenerateKeyParameters": {
+      "description": "The parameters used to regenerate an API key.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The id of the key to regenerate.",
+          "type": "string"
+        }
+      }
+    },
+    "GenerateSasTokenParameters": {
+      "description": "Parameters used for generating SAS token.",
+      "required": [
+        "resourceType",
+        "exp",
+        "sasKind"
+      ],
+      "type": "object",
+      "properties": {
+        "resourceType": {
+          "description": "The data plane resource type that the SAS token is authorized to access.",
+          "type": "string",
+          "enum": [
+            "kv",
+            "snapshot"
+          ],
+          "x-ms-enum": {
+            "name": "resourceType",
+            "modelAsString": true
+          }
+        },
+        "signedParameters": {
+          "description": "The parameters to define the data plane resource scope.",
+          "$ref": "#/definitions/SignedParameters"
+        },
+        "exp":{
+          "description": "The time that the SAS token expires in the Universal ISO 8601 DateTime format. Max allowed expiration is 1 year from the time of token creation.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "cch":{
+          "description": "Time (in seconds) for which the response may be cached by clients.",
+          "type": "number"
+        },
+        "sasKind":{
+          "description": "The kind of the SAS token.",
+          "type": "string",
+          "enum": [
+            "Primary",
+            "Secondary"
+          ],
+          "x-ms-enum": {
+            "name": "sasKind",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "SignedParameters": {
+      "description": "The parameters to define the data plane resource scope that the SAS token is authorized to access.",
+      "type": "object",
+      "properties": {
+        "key": {
+          "description": "A filter used to match keys. Syntax reference:\nhttps://aka.ms/azconfig/docs/keyvaluefiltering",
+          "type": "string"
+        },
+        "label": {
+          "description": "A filter used to match labels. Syntax reference:\nhttps://aka.ms/azconfig/docs/keyvaluefiltering",
+          "type": "string"
+        },
+        "tags":{
+          "description": "Am array of tag filters used to match tags.",
+          "type": "array",
+          "items": {
+            "description": "A filter used to match tag.",
+            "type": "string"
+          }
+        },
+        "name":{
+          "description": "The name of the snapshot. This is only applicable when the resourceType is snapshot.",
```

Comment:
Can you add the same description about resourceType restriction in key/label/tags